frontend_v2/src/app/components/challenge/challengediscuss/challengediscuss.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="challenge-card">
+  <div class="ev-card-panel card-bt-margin">
+    <div class="ev-md-container ev-panel-title">
+      <div class="row row-lr-margin">
+        <div class="col-sm-12 col-xs-12 col-lr-pad">
+          <h5 class="fw-light">Discuss</h5>
+          <span class="fs-16">Here you can discuss anything related to the <b>{{challenge.title}}</b>.Get all the anwers
```

Comment:
Minor: Hey @Kajol-Kumari  can we give a space here after full-stop (`.`) before the text `Get all the anwers for your queries, discuss any new idea......`

Reply:
Hey @Kajol-Kumari here we both missed this type error, please correct `anwers` to `answers`.

Reply:
Thanks @Sanji515 for pointing this!